src/components/Sections/SectionsView.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { SectionsViewProps } from 'components/Sections/SectionsTypes'
+import { SectionView } from 'components/Sections/SectionView'
+
+export const SectionsView = ({ data }: SectionsViewProps): JSX.Element => {
+  return (
+    <>
```

Comment:
You can return directly the content of the data?.map() ?

Reply:
true, but I thought it was cleaner to return a single jsx element with the fragment. otherwise we return an array of jsx element or even undefined elements.